### DISCUSSION TITLE
Configuring FullCalendar Plugin for Different Pages in Laravel

### DISCUSSION THREAD

**Discussion by hazhayder** [saade/filament-fullcalendar#132] [Q&A]
Hi there,

I'm currently working with the FullCalendar plugin in a Laravel project, and I'm facing an issue with configuring it differently for each widget on different pages. The plugin documentation suggests that the config can only be set in the AdminPanelProvider, which is limiting my ability to customize the FullCalendar settings for individual pages.

Here's a brief overview of the problem:

**Issue**:
The FullCalendar plugin allows config settings to be defined in the AdminPanelProvider, which makes it challenging to have different configurations for distinct pages or widgets.

**Desired Outcome:**
I would like to configure the FullCalendar plugin differently for each widget on various pages within my Laravel application.

**Question:**
How can I achieve page-specific configuration for the FullCalendar plugin when the documentation indicates that the config should be set at the AdminPanelProvider level? Is there a recommended workaround or best practice for achieving this level of customization on a per-widget basis?

Any guidance or code examples would be greatly appreciated!

**Comment by sitenzo** [comment]
@hazhayder See [Pull #130](https://github.com/saade/filament-fullcalendar/pull/130)
